### ISSUE TITLE
0000574:
refactor json frontend classes

### ISSUE THREAD

**Issue by Gloirin** [tine20/Tine-2.0-Open-Source-Groupware-and-CRM#378]
**Reported by pschuele on 13 Nov 2008 10:41**

use functions from Tinebase_Application_Frontend_Json_Abstract in all application json frontends

**Comment by pschuele** [comment]
imho no longer relevant

**Comment by Gloirin** [comment]
Depends on #379

**Comment by Gloirin** [comment]
Depends on #380

**Comment by Gloirin** [comment]
Depends on #381

**Comment by Gloirin** [comment]
Depends on #383
